src/test/testMocking.ts

```diff
@@ -61,6 +64,7 @@ describe('Mock', () => {
       returnListOfListOfIntArg(l: Int): [[Int]]
       returnListOfListOfObject: [[Bird!]]!
       returnStringArgument(s: String): String
+      node(id:String!):Flying
```

Comment:
the whole idea of this PR is that, when you run `node(id:'bee:123')` query, the mock server won't randomly choose a `Flying` type (could be either `Bee` or `Bird` ) but will run your mock of `Flying` code (where you specify whether to return a `bee` or `bird` ). It basically enables user to implement a `id-fetcher` for their node interface (which is where this PR comes from)